packer.json

```diff
@@ -4,12 +4,16 @@
     "build_region": "eu-west-1",
     "regions_to_copy_to": "eu-central-1",
     "branch": "master",
-    "max_spot_price": "0.1"
+    "max_spot_price": "0.1",
```

Comment:
Might be worth making this `0.2` as we had to wait a while last time for a `0.1` to show up